doc/version_history.rst

```diff
@@ -11,6 +11,7 @@ v0.5.0
 
 Rewrote the generation of the ts_xml XML files consolidating the commands and adding events.
 Rewrote the CSC and unit tests to take into account the changes in the commands and the added events.
+Removed trailing word 'perc' from command and telemetry enum items with a 'percentage' unit and sorted them alphabetically again.
```

Comment:
This seems to be a leading word (prefix), not a trailing word.

Also the primary driver for this update was to accommodate a change in ts_xml. I suggest saying so and specifying the new required version of ts_xml. You'll have to guess what the next release will be called, but it's crucial that users know this software is no longer compatible with ts_xml v9.0.

It is also useful to know that you changed entries in enum classes, because they are public classes, but I would argue that is less important than the fact that you updated your code for a new version of ts_xml with nicer field names.